src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -220,7 +224,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             'match' => false,
                         ]),
                         new Assert\NotBlank(),
-                        new Assert\Length(['min' => 3, 'max' => 128]),
+                        new Assert\Length([
+                            'max' => static::MAX_PRODUCT_NAME,
+                            'min' => static::MIN_PRODUCT_NAME,
+                            'maxMessage' => $this->translator->trans(
```

Comment:
Hi @yanmakouf 

although I'm sure this fixes the bug there's something weird behind the issue. It seems like it's trying to use a wording with plural form You can see in this string `This value is too long. It should have 128 character or less.|This value is too long. It should have 128 characters or less.` that is composed of a singular and a plural syntaxes separated by a `|`

This is how plural wordings were handled until Symfony 3.4 but I believe in Symfony 4.4 it doesn't work this way anymore. Your fix will work for this case but it won't fix the underlying issue, that we somehow use a wording with plural form which is not compatible with the framework anymore.

But the weirdest part is that this wording is supposed to come from the native constraint from Symfony, so we would expect them to be correctly formatted. This is worth digging a little but more to find the root cause.

Besides, I found a reference of this wording in this validator:
https://github.com/PrestaShop/PrestaShop/blob/c3202b242ceb24c1fbf367b74ff29dc5c3b0fbe5/src/PrestaShopBundle/Form/Validator/Constraints/TinyMceMaxLengthValidator.php#L87

So it's likely to have the same kind of bug, and potentially all fields that use this same `Assert\Length` constraint.

Reply:
The validator is supposed to handle the pluralization correctly based on the max option https://github.com/symfony/symfony/blob/8a62b40001ab80e32339fcf9fc5502ad460c68aa/src/Symfony/Component/Validator/Constraints/LengthValidator.php#L79 so maybe something is wrong in our Translator configuration and it is not able to understand plural forms anymore.

Reply:
> This is how plural wordings were handled until Symfony 3.4 but I believe in Symfony 4.4 it doesn't work this way anymore. Your fix will work for this case but it won't fix the underlying issue, that we somehow use a wording with plural form which is not compatible with the framework anymore.

I agree we need to explore more but as @yanmakouf started working on prestashop yesterday 😅 I'm afraid asking him to dive deep into the translator is not a good welcome gift so maybe he will XP a little on other topics before coming back to this one, better prepared 💪

Reply:
Yes I definitely agree with you. I noticed it before hand but thought I'd drop a quick fix in a first commit, and then later on, as Mthieu said, I will come back to try'n fix the core issue here (probably translations related as you said).